schema.sql

```diff
@@ -31,4 +31,30 @@ ALTER TABLE animals DROP COLUMN species;
 
 ALTER TABLE animals ADD species_id integer, ADD FOREIGN KEY (species_id) REFERENCES species(id);
 
-ALTER TABLE animals ADD owner_id integer, ADD FOREIGN KEY (owner_id) REFERENCES owners(id);
\ No newline at end of file
+ALTER TABLE animals ADD owner_id integer, ADD FOREIGN KEY (owner_id) REFERENCES owners(id);
+
+-- Day4
+
+CREATE TABLE vets (
+    id integer PRIMARY KEY,
+    name VARCHAR(150) NOT NULL,
+    age integer NOT NULL,
+    date_of_graduation date NOT NULL
+);
+
+CREATE TABLE specializations (
+    id integer PRIMARY KEY,
```

Comment:
- [ ] The same thing with this table, the id should be autoincremented PRIMARY KEY. Take a look at the comment above to learn how to do that.